scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -58,12 +67,51 @@ trait BaseReplState {
     useHdfsMode_()
     customConfig -= mr1Key
     customConfig -= mr2Key
+    printModeBanner()
   }
 
   def useHdfsLocalMode() {
     useHdfsMode_()
     customConfig += mr1Key -> mrLocal
     customConfig += mr2Key -> mrLocal
+    printModeBanner()
+  }
+
+  private[scalding] def printModeBanner(): Unit = {
+    val (modeString, homeDir) = mode match {
+      case localMode: Local => {
+        (localMode.toString, System.getProperty("user.dir"))
+      }
+      case hdfsMode: Hdfs => {
+        val defaultFs = FileSystem.get(hdfsMode.jobConf)
+        val m = customConfig.get(mr2Key) match {
+          case Some("local") =>
+            s"${hdfsMode.getClass.getSimpleName}Local(${hdfsMode.strict})"
+          case _ =>
+            s"${hdfsMode.getClass.getSimpleName}(${hdfsMode.strict})"
+        }
+        (m, defaultFs.getWorkingDirectory.toString)
```

Comment:
Is the working directory for the default `FileSystem` always the user's home directory?